hnn_core/viz.py

```diff
@@ -210,7 +210,7 @@ def plot_extracellular(times, data, tmin=None, tmax=None, ax=None,
 
 
 def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
-                color=None, show=True):
+                color=None, average=True, show=True):
```

Comment:
```suggestion
                color=None, average=False, show=True):
```

I think this is a better default for two reasons: 1) it keeps the existing plots the same with the same color etc., 2) We probably don't want to see the average unless there are at least a few dipoles